modules/rosa-policy-identity-access-management.adoc

```diff
@@ -43,7 +43,7 @@ Each of these access types have different levels of access to components:
 == SRE access to AWS accounts
 Red Hat personnel do not access AWS accounts in the course of routine {product-title} operations. For emergency troubleshooting purposes, the SREs have well-defined and auditable procedures to access cloud infrastructure accounts.
 
-SREs generate a short-lived AWS access token for the `BYOCAdminAccess` user using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. The `BYOCAdminAccess` user has the AdministratorAccess IAM policy attached.
+SREs generate a short-lived AWS access token for a reserved role using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached and is used for administration. For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached and is used for administration. (Both STS and non-STS clusters use the AWS STS service for SRE access.)
```

Comment:
* `AdministratorAccess` should be in backticks (like the other role is)
* The parentheses around the last sentence should be removed. Also, would it make sense to put this sentence before the sentences that start with "For non-STS..." and "For STS..."?

Reply:
done, all three